refactor(cijoe/bench): move shared functionality to imported script

NaddiNadja opened the pull request:
Bump cijoe to latest version and refactor shared functionality into imported script.

Our two workflows "latency" and "bench" that each generate graphs and reports, shared functionality in the plotting and reporting path of the workflows. This funcationliaty could not be put into a shared script before cijoe support the functionality of importing self-made scripts, which was added in v0.9.35.

NaddiNadja committed:
make(cijoe): bump to v0.9.41

Signed-off-by: Nadja Brix Koch <n.koch@samsung.com>

NaddiNadja committed:
refactor(cijoe/bench): move shared functionality to imported script

The 'latency' and 'bench' workflows, which generate graphs and
reports, share functionality in the plotting and reporting paths. This
functionality could not be moved to a shared script until Cijoe added
support for importing self-made scripts in version 0.9.35.

Signed-off-by: Nadja Brix Koch <n.koch@samsung.com>

NaddiNadja commented:
This is waiting for the next version of cijoe (v0.9.40+), where we've fixed that calling `powershell "...."` hangs forever.

safl reviewed (dismissed):
LGTM!

karlowich commented on cijoe/scripts/plotter.py:
```diff
@@ -0,0 +1,235 @@
+from cijoe.core.resources import dict_from_yamlfile
+
+OUTPUT_NORMALIZED_FILENAME = "benchmark-output-normalized.json"
+PLOT_SCALE = 1.5
```
It seems that two scripts import this, but the original value is not 1.5 for both. `latency_plotter` originally had 1.3.
Is this a problem?

NaddiNadja replied:
No, it only defines how much "air" there is above the bars in the plot. 1.5 looks fine, in my opinion

karlowich commented on cijoe/scripts/reporter.py:
```diff
@@ -1,64 +1,71 @@
-#!/usr/bin/env python3
-"""
-Report templates
-============================
-
-Create the general templates for the xNVMe reports.
-
-"""
```
Is there a need to remove the the doc-string?

NaddiNadja commented:
@safl and @karlowich , can you take a look again? The only things changed is that I added docs to the `cijoe/scripts/reporter.py` script, and that I've added the `--monitor` argument for all our cijoe runs in gha :)

karlowich reviewed (dismissed):
LGTM

safl reviewed (approved):
LGTM!
